frontend/trpc/server.ts

```
@@ -24,6 +24,7 @@ import { quickbooksRouter } from "./routes/quickbooks";
 import { shareHoldingsRouter } from "./routes/shareHoldings";
 import { tenderOffersRouter } from "./routes/tenderOffers";
 import { usersRouter } from "./routes/users";
+import { companyInviteLinksRouter } from "./routes/companyInviteLinks";
```

ESLint

[error] `./routes/companyInviteLinks` import should occur before import of `./routes/companyUpdates`

(import/order)